ban/http/api.py

```diff
@@ -80,6 +81,10 @@ def get_object(self, identifier):
                 link(headers, uri, 'alternate')
                 choices.append(uri)
             abort(300, headers=headers, choices=choices)
+        except IsDeletedError as err:
+            if request.method not in ['GET', 'PUT']:
+                abort(410, error='Resource `{}` is deleted'.format(identifier))
```

Comment:
That's two different stacks: `coerce` and `mark_deleted`. We still want to raise an error from `coerce` :)